va/va_dec_av1.h

```diff
@@ -0,0 +1,546 @@
+            uint32_t        disable_cdf_update                          : 1;
+            uint32_t        allow_screen_content_tools                  : 1;
+            uint32_t        force_integer_mv                            : 1;
+            uint32_t        allow_intrabc                               : 1;
+            uint32_t        use_superres                                : 1;
+            uint32_t        allow_high_precision_mv                     : 1;
+            uint32_t        is_motion_mode_switchable                   : 1;
+            uint32_t        use_ref_frame_mvs                           : 1;
+            /* disable_frame_end_update_cdf is coded as refresh_frame_context. */
+            uint32_t        disable_frame_end_update_cdf                : 1;
+            uint32_t        uniform_tile_spacing_flag                   : 1;
+            uint32_t        allow_warped_motion                         : 1;
+            /** \brief indicate if current frame in large scale tile mode */
+            uint32_t        large_scale_tile                            : 1;
+            uint32_t        reserved                                    : 15;
+        } bits;
+        uint32_t            value;
+    } pic_info_fields;
+
+
+    /** \brief Supper resolution scale denominator.
+     *  value range [9..16]
+     */
+    uint8_t                 superres_scale_denominator;
+
+    /** \brief Interpolation filter.
+     *  value range [0..9]
+     */
+    uint8_t                 interp_filter;
+
+    /** \brief luma loop filter levels.
+     *  value range [0..63].
+     */
+    uint8_t                 filter_level[2];
+
+    /** \brief chroma loop filter levels.
+     *  value range [0..63].
+     */
+    uint8_t                 filter_level_u;
+    uint8_t                 filter_level_v;
+
+    union
+    {
+        struct
+        {
+            /** \brief flags for reference pictures
+             *  same syntax and semantic as those in AV1 code
+             */
+            uint8_t         sharpness_level                             : 3;
+            uint8_t         mode_ref_delta_enabled                      : 1;
+            uint8_t         mode_ref_delta_update                       : 1;
+            uint8_t         reserved                                    : 3;
+        } bits;
+        uint8_t             value;
+    } loop_filter_info_fields;
+
+    /** \brief The adjustment needed for the filter level based on
+     *  the chosen reference frame.
+     *  value range [-63..63].
+     */
+    int8_t                  ref_deltas[8];
+
+    /** \brief The adjustment needed for the filter level based on
+     *  the chosen mode.
+     *  value range [-63..63].
+     */
+    int8_t                  mode_deltas[2];
+
+    /** \brief quantization
+     */
+    /** \brief Y AC index
+     *  value range [0..255]
+     */
+    uint16_t                base_qindex;

```

Comment:
why we nedd uint16_t if we know it in [0.. 255]

Reply:
I will fix it, thanks